week-1-practise/level-1/8-function-output-1.js

```diff
@@ -1,12 +1,13 @@
 function getNumber() {
-    return Math.random() * 10;
+    return (Math.random() * 10);
```

Comment:
You aren't wrong, but the extra parentheses (`(` and `)`) here are kind of unnecessary.

Sometimes when we're doing maths with our code we want to add extra parentheses to make it easier to read, but in this case, the extra parens don't really add anything.